.evergreen/config_generator/components/funcs/find_cmake_latest.py

```diff
@@ -0,0 +1,25 @@
+from shrub.v3.evg_command import EvgCommandType
+
+from config_generator.etc.function import Function
+from config_generator.etc.utils import bash_exec
+
+
+class FindCMakeLatest(Function):
+    name = 'find-cmake-latest'
+    command_type = EvgCommandType.SETUP
+    commands = [
+        bash_exec(
+            command_type=command_type,
+            retry_on_failure=True,
```

Comment:
Evergreen docs [note](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files) "the number of times a single task can be automatically restarted on failure is limited to 1 time".

Was implementing a retry within the download script considered (e.g. adding `--retry` to the `curl` command)?

Reply:
The [download-file](https://github.com/mongodb/mongo-c-driver/commit/5c101ce5a565ffa9d160b6e44fb276b502b57157#diff-52db4cda561258ec41fbeb73717e37854bf6390c74d276e55058d49bc0781505R200) Bash function contains [--retry 5](https://github.com/mongodb/mongo-c-driver/blob/1761c7a5fd19c47b02779238a14159e9e8d94a32/tools/download.sh#L50) along with other recommended curl flags (moved from this file in https://github.com/mongodb/mongo-c-driver/commit/5c101ce5a565ffa9d160b6e44fb276b502b57157#diff-52db4cda561258ec41fbeb73717e37854bf6390c74d276e55058d49bc0781505R200, which is also when the obsolete `curl_args` variable was left behind). The `retry_on_failure: true` option is in hopes that a task-level restart may have a better/another opportunity to flush out spurious network failures.